File: JAVA LEC/L15/P1.java
//WAP to read list of marks and find sum,avg,max and min

import java.util.*;
import java.io.*;
class P1
{
public static void main(String args[])
{
Console c=System.console();
ArrayList<Integer> data=new ArrayList();

String ans=c.readLine("do u want to add element yes/no ");
while(ans.equals("yes"))
{
int d=Integer.parseInt(c.readLine("enter the data "));
data.add(d);
ans=c.readLine("do u want to add more elements yes/no ");
}
System.out.println("elements = "+data);
Collections.sort(data);

System.out.println("elements = "+data);

}
}

File: Python/python/L5/p1.py
#wapp to read the list of int from the user and display on screen

data=[]
ans=input("do u want to enter data y/n ")
while ans =='y':
	ele=int(input("enter element "))
	data.append(ele)
	ans=input("do u want to enter data y/n ")
print (data)
	
for d in data:
	print(d, end=' ')
print()
for i in range(len(data)):
	print(data[i],end=' ')
print()